lib/models/src/main/java/org/sagebionetworks/repo/model/table/ViewScopeFilter.java

```diff
@@ -0,0 +1,65 @@
+package org.sagebionetworks.repo.model.table;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+
+import org.sagebionetworks.repo.model.ObjectType;
+
+/**
+ * DTO that specifies the scope filter to apply when working with a view
+ * 
+ * @author Marco Marasca
+ */
+public class ViewScopeFilter {
+
+	private final ObjectType objectType;
+	private final List<Enum<?>> subTypes;
+	private final boolean filterByObjectId;
+	private final Set<Long> containerIds;
+
+	public ViewScopeFilter(ObjectType objectType, List<Enum<?>> subTypes, boolean filterByObjectId, Set<Long> containerIds) {
+		this.objectType = objectType;
+		this.subTypes = subTypes;
+		this.filterByObjectId = filterByObjectId;
+		this.containerIds = containerIds;
+	}
+	
+	public ObjectType getObjectType() {
+		return objectType;
+	}
+
+	public List<Enum<?>> getSubTypes() {

```

Comment:
Consider a List<String> here since at this level it does not matter that they were enums.